src/test/regress/expected/multi_colocated_shard_rebalance.out

```diff
@@ -412,8 +421,9 @@ ORDER BY s.shardid, sp.nodeport;
 (3 rows)
 
 -- try to move shard from wrong node
-SELECT master_move_shard_placement(13000021, 'localhost', :worker_1_port, 'localhost', :worker_2_port, 'force_logical');
-ERROR:  source placement must be in active state
+SELECT master_move_shard_placement(13000021, 'localhost', :master_port, 'localhost', :worker_1_port, 'force_logical');
```

Comment:
moves from nodes that don't have the shards are now allowed if the target does have the shards, so changed the ports